scripts/quarantine.yaml

```diff
@@ -67,3 +69,34 @@
   platforms:
     - nrf54l15dk/nrf54l15/cpuapp
   comment: "https://nordicsemi.atlassian.net/browse/NRFX-7878"
+
+- scenarios:
+    - sample.find_my.*
+    - test.find_my.*
+  platforms:
+    - nrf54h20dk@0.9.0/nrf54h20/cpuapp
+  comment: "Needs alignment to work with IronSide SE"
+
+- scenarios:
+    - application.jwt.*
+  platforms:
+    - nrf54h20dk@0.9.0/nrf54h20/cpuapp
+  comment: "Needs OTP service to be implemented in IronSide SE"
+
+- scenarios:
+    - sample.bluetooth.fast_pair.*
+  platforms:
+    - nrf54h20dk@0.9.0/nrf54h20/cpuapp
+  comment: "Needs alignment to work with IronSide SE"
+
+- scenarios:
+    - applications.nrf_desktop.*
+  platforms:
+    - nrf54h20dk@0.9.0/nrf54h20/cpuapp
+  comment: "Needs alignment to work with MCUBoot"
```

Comment:
Is the current state of quarantine the end state for this PR?

@sveinaga, is it okay for these projects to be quarantined? We may need to keep them in quarantine for the upcoming release, as we did not plan any work to address this migration. CC: @pdunaj

Reply:
Yes that was the desicion. 

@sveinaga and @pdunaj are both aware of this situation and there are tasks for enabling all of them again.